cpp/ql/src/change-notes/2024-01-29-uninitialized-local-false-positive.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Corrected a false positive with `cpp/uninitialized-local`: `a->func()` is a false positive if `func` is static regardless of if `a` is initializeed.
```

Comment:
I think this is slightly more clear since it actually says that we no longer flags up these calls.

```suggestion
* The "Potentially uninitialized local variable" query (`cpp/uninitialized-local`) no longer reports an alert when the local variable is used as a qualifier to a static member function call.
* ```